qiskit_optimization/applications/__init__.py

```diff
@@ -63,7 +65,8 @@
 from .vertex_cover import VertexCover
 from .sk_model import SKModel
 
-__all__ = [
+_all__ = [
```

Comment:
You should not overwrote the fix.
```suggestion
__all__ = [
```

Reply:
thank you!